packages/realm-tools/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@realm/tools",
+  "version": "0.0.1",
+  "description": "Tools for work with Realm",
+  "bin": {
+    "realm-schema": "./dist/realm-schema.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "lint": "eslint --ext .js,.ts .",
+    "prepare": "rm -rf dist && npm run build",
+    "start": "ts-node src/cli.ts",
```

`cli.ts` does not exist
```suggestion
    "start": "ts-node src/realm-schema.ts",
```

---

> Is the idea to have multiple tools in the future that live under the same cli command?

No, and I will remove `start`.